chord_metadata_service/mohpackets/api_discovery.py

```diff
@@ -120,3 +142,196 @@ class DiscoveryBiomarkerViewSet(DiscoveryMixin, BaseBiomarkerViewSet):
+#        CUSTOM OVERVIEW API ENDPOINTS        #
+#                                             #
+###############################################
+
+
+@extend_schema(
+    description="MoH cohorts count",
+    responses={
+        200: inline_serializer(
+            name="moh_overview_cohort_count_response",
+            fields={
+                "cohort_count": serializers.IntegerField(),
+            },
+        )
+    },
+)
+@api_view(["GET"])
+@throttle_classes([MoHRateThrottle])
+def cohort_count(_request):
+    """
+    Return the number of cohorts in the database.
+    """
+    return Response({"cohort_count": Program.objects.count()})
+
+
+@extend_schema(
+    description="MoH patients per cohort count",
+    responses={
+        200: inline_serializer(
+            name="moh_overview_patient_per_cohort_response",
+            fields={
+                "patients_per_cohort_count": serializers.IntegerField(),
+            },
+        )
+    },
+)
+@api_view(["GET"])
+@throttle_classes([MoHRateThrottle])
+def patient_per_cohort_count(_request):
+    """
+    Return the number of patients per cohort in the database.
+    """
+    cohorts = Donor.objects.values_list("program_id", flat=True)
+    return Response(count_terms(cohorts))
+
+
+@extend_schema(
+    description="MoH individuals count",
+    responses={
+        200: inline_serializer(
+            name="moh_overview_individual_count_response",
+            fields={
+                "individual_count": serializers.IntegerField(),
+            },
+        )
+    },
+)
+@api_view(["GET"])
+@throttle_classes([MoHRateThrottle])
+def individual_count(_request):
+    """
+    Return the number of individuals in the database.
+    """
+    return Response({"individual_count": Donor.objects.count()})
+
+
+@extend_schema(
+    description="MoH gender count",
+    responses={
+        200: inline_serializer(
+            name="moh_overview_gender_count_response",
+            fields={
+                "gender_count": serializers.IntegerField(),
+            },
+        )
+    },
+)
+@api_view(["GET"])
+@throttle_classes([MoHRateThrottle])
+def gender_count(_request):
+    """
+    Return the count for every gender in the database.
+    """
+    genders = SampleRegistration.objects.values_list("gender", flat=True)
+    return Response(count_terms(genders))
+
+
+@extend_schema(
+    description="MoH cancer types count",
+    responses={
+        200: inline_serializer(
+            name="moh_overview_cancer_type_count_response",
+            fields={
+                "cancer_type_count": serializers.IntegerField(),
+            },
+        )
+    },
+)
+@api_view(["GET"])
+@throttle_classes([MoHRateThrottle])
+def cancer_type_count(_request):
+    """
+    Return the count for every cancer type in the database.
+    """
+    cancer_types = Donor.objects.values_list("primary_site", flat=True)
+    return Response(count_terms(cancer_types))
+
+
+@extend_schema(
```

Comment:
if you don't want to repeat this for each endpoint, you can create a generic serializer that returns count only, like what I did with DiscoverySerializer